src/webparts/sendToLaserficheLoginComponent/components/SendToLaserficheLoginComponent.tsx

```diff
@@ -39,6 +40,7 @@ const YOU_MUST_BE_CLOUD_USER_TO_USE_WEB_PART =
   'You must be a currently licensed Laserfiche Cloud user to use this web part.';
 const FOR_MORE_INFO_VISIT = 'For more information visit';
 
+const needLaserficheSignInPage = `Sign in failed. Missing ${LASERFICHE_SIGNIN_PAGE_NAME} SharePoint page. Please refer to the Adding App to SharePoint Site topic in the administration guide for configuration steps.`;

```

Comment:
is it important to have the "Sign in failed" message? If not, we could use the exact same message as in the other webparts